url/urlsearchparams-constructor.html

```diff
@@ -18,15 +18,25 @@
 }, 'Basic URLSearchParams construction');
 
 test(function() {
-    assert_throws(new TypeError(), function () { URLSearchParams(); },
-                  'Calling \'URLSearchParams\' without \'new\' should throw.');
-    assert_throws(new TypeError(), function () { new URLSearchParams(DOMException.prototype); });
-    var params = new URLSearchParams('');
+    var params = new URLSearchParams()
+    assert_equals(params.toString(), "")
+}, "URLSearchParams constructor, no arguments")
+
+test(() => {
+    params = new URLSearchParams(DOMException.prototype);
+    assert_equals(params.toString(), "Error=")
+}, "URLSearchParams constructor, DOMException.prototype as argument")
+
+test(() => {
+    params = new URLSearchParams('');
     assert_true(params != null, 'constructor returned non-null value.');
     assert_equals(params.__proto__, URLSearchParams.prototype, 'expected URLSearchParams.prototype as prototype.');
+}, "URLSearchParams constructor, empty string as argument")
+
+test(() => {
     params = new URLSearchParams({});
     assert_equals(params + '', '%5Bobject+Object%5D=');
```

Comment:
I think this should now be equivalent to no arguments per record semantics. So just empty string.